### ISSUE TITLE
ccxt throws wrong exception when FTX is unavailable

### ISSUE THREAD

**Issue by mak1986** [ccxt/ccxt#8142]
Currently ccxt throws ExchangeError when FTX exchange is unavailable. According to the documentation it should throw NetworkError (ExchangeNotAvailable)

- OS: macOS Mojave
- Programming Language version: Python 3.6.9
- CCXT version: 1.37.12 (I've upgraded to the latest but just asking in case this bug hasn't been fixed.)

```
    def get_open_orders(self, symbol=None):
        if symbol is None:
            symbol = self._config['exchange']['symbol']

        return self.request(self._ex.fetch_open_orders, symbol)
        
    def request(self, action, *args):

        try:
            return action(*args)

        # ExchangeError

        except ccxt.InsufficientFunds as e:
            traceback_error_str = traceback.format_exception(*sys.exc_info())
            self._logger.warning(
                f"Insufficient funds error: {str(action)} {str(args)}, {type(e).__name__} {str(e)} {traceback_error_str}")
            return self.retry(action, 100, 1, *args)
        except ccxt.ExchangeError as e:
            traceback_error_str = traceback.format_exception(*sys.exc_info())
            self._logger.error(
                f"Exchange error: {str(action)} {str(args)}, {type(e).__name__} {str(e)} {traceback_error_str}")
            raise SystemExit(1)

        # NetworkError

        except ccxt.NetworkError as e:
            traceback_error_str = traceback.format_exception(*sys.exc_info())
            self._logger.warning(
                f"Network error: {str(action)} {str(args)}, {type(e).__name__} {str(e)} {traceback_error_str}")
            return self.retry(action, 300, 1, *args)
```

```
 [
   "Traceback (most recent call last):\n",
   "  File \"/home/app/.local/lib/python3.6/site-packages/ccxt/base/exchange.py\", line 588, in fetch\n    response.raise_for_status()\n",
   "  File \"/home/app/.local/lib/python3.6/site-packages/requests/models.py\", line 941, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\n",
   "requests.exceptions.HTTPError: 503 Server Error: SERVICE UNAVAILABLE for url: https://ftx.com/api/orders?market=TRX%2FUSD\n",
   "\nDuring handling of the above exception, another exception occurred:\n\n",
   "Traceback (most recent call last):\n",
   "  File \"/home/app/makabot/exchanges/BaseExchange.py\", line 159, in request\n    return action(*args)\n",
   "  File \"/home/app/.local/lib/python3.6/site-packages/ccxt/ftx.py\", line 1162, in fetch_open_orders\n    response = getattr(self, method)(self.extend(request, query))\n",
   "  File \"/home/app/.local/lib/python3.6/site-packages/ccxt/base/exchange.py\", line 465, in inner\n    return entry(_self, **inner_kwargs)\n",
   "  File \"/home/app/.local/lib/python3.6/site-packages/ccxt/base/exchange.py\", line 490, in request\n    return self.fetch2(path, api, method, params, headers, body)\n",
   "  File \"/home/app/.local/lib/python3.6/site-packages/ccxt/base/exchange.py\", line 486, in fetch2\n    return self.fetch(request[\\'url\\'], request[\\'method\\'], request[\\'headers\\'], request[\\'body\\'])\n",
   "  File \"/home/app/.local/lib/python3.6/site-packages/ccxt/base/exchange.py\", line 604, in fetch\n    self.handle_errors(http_status_code, http_status_text, url, method, headers, http_response, json_response, request_headers, request_body)\n",
   "  File \"/home/app/.local/lib/python3.6/site-packages/ccxt/ftx.py\", line 1508, in handle_errors\n    raise ExchangeError(feedback)  # unknown message\n",
   "ccxt.base.errors.ExchangeError: ftx {\"error\":\"Please retry request\",\"success\":false}\n\n"
]
```

**Comment by kroitor** [comment]
Thanks for reporting! The new build will arrive in 20 minutes.

**Comment by mak1986** [comment]
Thank you for your quick response. :)

**Comment by kroitor** [comment]
This is now fixed in 1.39.52+, let us know if you have any other difficulties with it! Thx again!

**Comment by mak1986** [comment]
Hi again. I just saw today that Ftx also has another retry error message "Please try again".  

```
2020-12-30 13:24:54,301 | WARNING | BaseExchange:line-179 | bot_0109 | Network error: <bound method ftx.fetch_open_orders of ccxt.ftx()> ('TRX/USD',), ExchangeNotAvailable ftx {"error":"Please retry request","success":false}
 ['Traceback (most recent call last):\n', '  File "/home/app/.local/lib/python3.6/site-packages/ccxt/base/exchange.py", line 588, in fetch\n    response.raise_for_status()\n', '  File "/home/app/.local/lib/python3.6/site-packages/requests/models.py", line 943, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\n', 'requests.exceptions.HTTPError: 503 Server Error: SERVICE UNAVAILABLE for url: https://ftx.com/api/orders?market=TRX%2FUSD\n', '\nDuring handling of the above exception, another exception occurred:\n\n', 'Traceback (most recent call last):\n', '  File "/home/app/makabot/exchanges/BaseExchange.py", line 159, in request\n    return action(*args)\n', '  File "/home/app/.local/lib/python3.6/site-packages/ccxt/ftx.py", line 1284, in fetch_open_orders\n    response = getattr(self, method)(self.extend(request, query))\n', '  File "/home/app/.local/lib/python3.6/site-packages/ccxt/base/exchange.py", line 465, in inner\n    return entry(_self, **inner_kwargs)\n', '  File "/home/app/.local/lib/python3.6/site-packages/ccxt/base/exchange.py", line 490, in request\n    return self.fetch2(path, api, method, params, headers, body)\n', '  File "/home/app/.local/lib/python3.6/site-packages/ccxt/base/exchange.py", line 486, in fetch2\n    return self.fetch(request[\'url\'], request[\'method\'], request[\'headers\'], request[\'body\'])\n', '  File "/home/app/.local/lib/python3.6/site-packages/ccxt/base/exchange.py", line 604, in fetch\n    self.handle_errors(http_status_code, http_status_text, url, method, headers, http_response, json_response, request_headers, request_body)\n', '  File "/home/app/.local/lib/python3.6/site-packages/ccxt/ftx.py", line 1682, in handle_errors\n    self.throw_broadly_matched_exception(self.exceptions[\'broad\'], error, feedback)\n', '  File "/home/app/.local/lib/python3.6/site-packages/ccxt/base/exchange.py", line 509, in throw_broadly_matched_exception\n    raise broad[broad_key](message)\n', 'ccxt.base.errors.ExchangeNotAvailable: ftx {"error":"Please retry request","success":false}\n\n']
2020-12-30 13:24:54,302 | WARNING | BaseExchange:line-126 | bot_0109 | An exchange error occured, retrying in 1 second(s) (1. attempt)
2021-01-01 01:10:25,389 | ERROR | BaseExchange:line-171 | bot_0109 | Exchange error: <bound method ftx.create_order of ccxt.ftx()> ('TRX/USD', 'limit', 'buy', '18', '0.0268975'), ExchangeError ftx {"error":"Please try again","success":false}
 ['Traceback (most recent call last):\n', '  File "/home/app/.local/lib/python3.6/site-packages/ccxt/base/exchange.py", line 588, in fetch\n    response.raise_for_status()\n', '  File "/home/app/.local/lib/python3.6/site-packages/requests/models.py", line 943, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\n', 'requests.exceptions.HTTPError: 503 Server Error: SERVICE UNAVAILABLE for url: https://ftx.com/api/orders\n', '\nDuring handling of the above exception, another exception occurred:\n\n', 'Traceback (most recent call last):\n', '  File "/home/app/makabot/exchanges/BaseExchange.py", line 159, in request\n    return action(*args)\n', '  File "/home/app/.local/lib/python3.6/site-packages/ccxt/ftx.py", line 1036, in create_order\n    response = getattr(self, method)(self.extend(request, params))\n', '  File "/home/app/.local/lib/python3.6/site-packages/ccxt/base/exchange.py", line 465, in inner\n    return entry(_self, **inner_kwargs)\n', '  File "/home/app/.local/lib/python3.6/site-packages/ccxt/base/exchange.py", line 490, in request\n    return self.fetch2(path, api, method, params, headers, body)\n', '  File "/home/app/.local/lib/python3.6/site-packages/ccxt/base/exchange.py", line 486, in fetch2\n    return self.fetch(request[\'url\'], request[\'method\'], request[\'headers\'], request[\'body\'])\n', '  File "/home/app/.local/lib/python3.6/site-packages/ccxt/base/exchange.py", line 604, in fetch\n    self.handle_errors(http_status_code, http_status_text, url, method, headers, http_response, json_response, request_headers, request_body)\n', '  File "/home/app/.local/lib/python3.6/site-packages/ccxt/ftx.py", line 1683, in handle_errors\n    raise ExchangeError(feedback)  # unknown message\n', 'ccxt.base.errors.ExchangeError: ftx {"error":"Please try again","success":false}\n\n']
```
